share/man/kodev.1

```diff
@@ -0,0 +1,130 @@
+.TH KODEV 1
+.SH NAME
+kodev \- Kore project management tool
+
+.SH SYNOPSIS
+.BR kodev
+[\fIOPTION\fR] ...
+
+.SH DESCRIPTION
+This documentation describes the application management tools for building and
+running a kore project. For information regarding the technical interface and
+C bindings, look to
+.BR KORE(3)
+where these functions and structures are described.
+.BR kore
+projects may be managed using the following OPTIONS;
+
+.BR create
+.RS
+Create a new application skeleton with the name that is passed to it. This will
+create a new directory with all the files required to begin hacking about.
+This will also generate a self-signed certificate \fIunless\fR \fBkore\fR was
+built with \fBNOTLS=1\fR. Note this certificate should not be used for
```

Comment:
This isn't true.

Kodev will always generate the self-signed certs / write out the DH parameters unless single-binary is turned on in the build.conf and NOTLS=1 is set in the kore_flavor option.